numismatic/feeds/bravenewcoin.py

```diff
@@ -0,0 +1,33 @@
+from itertools import product
+import attr
+from .base import Feed
+from ..libs.config import get_config
+
+class BraveNewCoin(Feed):
+    config = get_config()['bravenewcoin']
+    api_key_id = (config.get('api_key_id', '') if config else '')
+    api_key_secret = (config.get('api_key_secret', '') if config else '')
```

Comment:
I think that these should be instance variables. You might want to have different values within the same session, say if you want to trade on two different accounts at the same time.